src/components/SettingsPage/Tools.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import Box, { Card } from 'components/base/Box'
+import Modal, { ModalBody } from 'components/base/Modal'
+import Button from 'components/base/Button'
+import QRCodeExporter from 'components/QRCodeExporter'
+
+class TabProfile extends PureComponent<*, *> {
+  state = {
+    qrcodeMobileExportModal: false,
+  }
+
+  onQRCodeMobileExport = () => {
+    this.setState({ qrcodeMobileExportModal: true })
+  }
+
+  onRequestClose = () => {
+    this.setState({ qrcodeMobileExportModal: false })
+  }
+
+  renderQRCodeModal = ({ onClose }: *) => (
+    <ModalBody
+      onClick={onClose}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
```

Comment:
`ModalBody` renders a `Box` so you can directly

```js
<ModalBody onClick={onClose} justify="center" align="center">
```

Reply:
cool